### PR TITLE
Add auto-sort feature into the UG and changed data file name accordingly

### PR DESCRIPTION
**Pull request by RezwanAhmed123**
The auto-sort feature while already implemented before, was not added to the user guide yet.

According to what was allowed in the v1.6 milestone, we changed the data file name from addressbook to insureance to fit our app better. Referring to point 2 below, this should be an allowed fix.

image

This was also updated in the user guide.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -401,7 +401,7 @@ App data are saved in the hard disk automatically after any command that changes
 
 ### Editing the data file
 
-App data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+App data are saved automatically as a JSON file `[JAR file location]/data/inSUREance.json`. Advanced users are welcome to update data directly by editing that data file.
```
Comment (matthewyip1511) [inline docs/UserGuide.md:404]: tiny nit here, but the JAR file uses uppercase `InSUREance.jar` while this is lowercase. Should we standardise to one of them (I am happy to change the name for JAR file too if we need to)

Comment (RezwanAhmed123) [reply docs/UserGuide.md:404]: lets standardise to `InSUREance`

Comment (RezwanAhmed123) [reply docs/UserGuide.md:404]: fixed

**Review by matthewyip1511 — APPROVED** [review]
LGTM, merging 👍

**Pull request merged**